ui/src/services/NodeVolumesUtils.test.js

```diff
@@ -4,6 +4,7 @@ import {
   STATUS_PENDING,
   STATUS_FAILED,
   STATUS_AVAILABLE,
```

Comment:
I think you can remove `STATUS_AVAILABLE` from the import here: it's no longer used in this file.